addons/website_sale/tests/test_website_sequence.py

```diff
@@ -21,7 +21,9 @@ def setUpClass(cls):
         cls.public_user = cls.env.ref('base.public_user')
 
         ProductTemplate = cls.env['product.template']
-        product_templates = ProductTemplate.search([])
+        # The "Service on Timesheet" product cannot be archived nor deleted
+        time_product = cls.env.ref('sale_timesheet.time_product').product_tmpl_id
```

Comment:
`website_sale` does not depend on `sale_timesheet`, you shouldn't strictly reference a product that might not exist.
This test now fails when you run it with only `website_sale` installed.